### PR TITLE
feat: Enable proxy: Hive connector

### LINKED ISSUE #517: Add Hive connector

### PR DESCRIPTION
**Pull request by sanjay-a01**
## Checklist
- [x] Ran Linter
- [x] Created PR from non-main branch 

## Notes
It supports pagination

## Catelog variables
Hive Provider = "hive"

### TOKEN RESPONSE
It is Non-Oauth2 Compliant

### GET
URL: https://proxy.withampersand.com/api/v1/workspaces/FhZP73NoDZ5Tfa7tr/projects?sortBy=name+asc
image

### POST
URL: https://proxy.withampersand.com/api/v1/actions/create
image

### PUT
URL: https://proxy.withampersand.com/api/v1/projects/MAzh7Go6gaC9djd3Q?user_id=chhC559rJBW3J3sYc
image

### DELETE
URL: https://proxy.withampersand.com/api/v1/projects/MAzh7Go6gaC9djd3Q
image

### Pagination
Requesting a paginated list of labels using limit:
URL: https://proxy.withampersand.com/api/v1/workspaces/FhZP73NoDZ5Tfa7tr/labels?limit=2
image

Requesting the next page using the limit and skip 
URL: https://proxy.withampersand.com/api/v1/workspaces/FhZP73NoDZ5Tfa7tr/labels?limit=2&skip=2
image

### Invalid request
image

### Success Console Operation
image

### Failure Console Operation
image

Closes #517

### CODE REVIEW

File: providers/hive.go [hunk 1]
```diff
@@ -17,10 +17,7 @@ func init() {
 		},
 		//nolint:lll
 		Media: &Media{
-			DarkMode: &MediaTypeDarkMode{
```
Comment (laurenzlong) [inline providers/hive.go:20]: Seems like you accidentally deleted the DarkMode logos, can you bring them back?

File: providers/hive.go [hunk 2]
```diff
@@ -30,10 +27,9 @@ func init() {
 			BulkWrite: BulkWriteSupport{
 				Insert: false,
 				Update: false,
-				Upsert: false,
```
Comment (laurenzlong) [inline providers/hive.go:33]: Same here, you accidentally removed this line

**Review by laurenzlong — APPROVED** [review]
Good to merge once you bring back the lines that were accidentally deleted

File: providers/hive.go (hunk 1 above)
Comment (sanjay-a01) [reply providers/hive.go:20]: Sorry, I'll add the DarkMode Logos and upsert in the BulkWriteSupport.

**Comment by sanjay-a01** [comment]
> Good to merge once you bring back the lines that were accidentally deleted

FYI, I have added the deleted lines

**Review by laurenzlong — APPROVED** [review]

**Pull request merged**